tests/e2e/core-tests/specs/shopper/front-end-my-account-pay-order.test.js

```diff
@@ -17,7 +17,6 @@ const simpleProductName = config.get( 'products.simple.name' );
 const runMyAccountPayOrderTest = () => {
 	describe('Customer can pay for their order through My Account', () => {
 		beforeAll(async () => {
-			await merchant.login();
 			simplePostIdValue = await createSimpleProduct();
 			await merchant.logout();
```

Comment:
We should be safe to remove this logout() call here.

Reply:
Done removing this unnecessary merchant logout.

Reply:
Thanks for taking care of this!